orca/service_test.go

```diff
@@ -126,7 +116,10 @@ func (s) TestE2E_CustomBackendMetrics_OutOfBand(t *testing.T) {
 				errCh <- fmt.Errorf("UnaryCall failed: %v", err)
 				return
 			}
-			time.Sleep(time.Millisecond)
+			mu.Lock()
```

Comment:
why did request counter move here? why can't it remain in stub function?

Reply:
By managing the counter outside the stub, we can safely increment it in a concurrent context. This will ensure that we accurately track how many requests have been sent, regardless of their processing order or timing.

Reply:
The counter needs to be updated inside of the `UnaryCall` and `t.smr.SetNamedUtilization(requestsMetricKey, float64(t.requests)*0.01)` line should remain like earlier. I don't see a point in hard-coding `requestsMetricKey` to `0.2`.